invokeai/backend/prompting/conditioning.py

```diff
@@ -71,6 +71,7 @@ def get_uc_and_c_and_ec(
         text_encoder=text_encoder,
         textual_inversion_manager=model.textual_inversion_manager,
         dtype_for_device_getter=torch_dtype,
+        truncate_long_prompts=False
```

Comment:
This doesn't appear to be used anywhere.

Reply:
this is in fact a necessary init argument for Compel...